pyomo/contrib/mindtpy/tests/eight_process_problem.py

```diff
@@ -144,6 +144,6 @@ def __init__(self, *args, **kwargs):
 
         """Bound definitions"""
         # x (flow) upper bounds
-        x_ubs = {3: 2, 5: 2, 9: 2, 10: 1, 14: 1, 17: 2, 19: 2, 21: 2, 25: 3}
+        x_ubs = {3: 2, 5: 2, 9: 2, 10: 1, 14: 1, 17: 2, 18: 1.4, 19: 2, 21: 2, 25: 3}
```

Comment:
Out of curiosity, was this bound in the original problem, or are you adding it to support testing with the lp/nlp solver?

Reply:
I tested the case again without `18: 1.4`, it made no difference to the results.
For the other two bound change in MINLP2_simple.py and MINLP3_simple.py, MindtPy is able to deal with these problems without the changes. However, some warnings will be reported if the bounds are not added.

Reply:
Since here it makes no difference, I will remove these.

Users will often submit models that do not have all bounds specified. @bernalde is it your intent to warn them and have them provide some bounding information, in that case? If so, it may be good to have some tests to that effect.

Reply:
Currently, the warning is raised by the MILP solvers when a variable is unbounded in the master problem (it may be bounded for the MINLP given the nonlinear constraints), and depending on each they add redundant bounds to avoid the problem being unbounded.
MindtPy by itself does not add those bounds since we do not know them a-priori. Maybe if we used some bound propagation (@michaelbynum) although I do not know what would be the efficiency effect of that.